src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
 public class PasswordValidator {
-    public boolean validate(String password, String repeatPassword) {
-        return false;
+    public static boolean validate(String password, String repeatPassword)
```

Comment:
```suggestion
    public boolean validate(String password, String repeatPassword)
```
static methods are usually a bad practice, let's not change it

Reply:
I know, but my IDEA say me - do this method static
https://prntscr.com/vvkw4g

Reply:
ok i fix this

Reply:
you just need to creat object to call non-static methods